identity/identity.go

```diff
@@ -6,49 +6,110 @@ import (
 
 	"github.com/ONSdigital/go-ns/rchttp"
 
+	"context"
 	"github.com/ONSdigital/go-ns/log"
+	"encoding/json"
+	"io/ioutil"
 )
 
+//go:generate moq -out identitytest/http_client.go -pkg identitytest . HttpClient
+
 var zebedeeURL = os.Getenv("ZEBEDEE_URL")
+var florenceHeaderKey = "X-Florence-Token"
```

Comment:
apart from zebedeeURL I think these are constants

Reply:
Changed to const